src/ocamlorg_frontend/components/header.eml

```diff
@@ -23,8 +23,9 @@ let render
 let search_dropdown () =
   <span id="header-search-indicator" class="mx-2 htmx-indicator">Searching...</span>
   <div id="header-search-results" aria-live="polite"></div>
-  <span class="pl-2 font-semibold">Or go to:</span>
-  <a class="flex py-2 px-4 gap-4 hover:bg-primary-100 font-semibold hover:font-semibold text-primary-600" href="<%s Url.api %>">
+  <hr>
+  <span class="pl-2 font-normal">Or go to</span>
```

Comment:
I have an idea here:

It probably makes sense to hide this span when we don't have any search results. What do you think?

Reply:
i tried to do this but it kept bring up some ui challenges because even with that span hidden the drop down background will always show. only way to implement this is to hide the dropdown div until at least one letter is typed on the input bar. unfortunately i was not able to do this

Reply:
the dropdown background showing should be fine because there's always the standard library link in it

Reply:
alright, i think my last commit has sorted this issue